song-server/src/main/java/bio/overture/song/server/service/AnalysisTypeService.java

```diff
@@ -246,9 +256,23 @@ private void validateAnalysisTypeSchema(JsonNode analysisTypeSchema) {
 
   @SneakyThrows
   private AnalysisType commitAnalysisType(
-      @NonNull String analysisTypeName, @NonNull JsonNode analysisTypeSchema) {
+      @NonNull String analysisTypeName,
+      @NonNull JsonNode analysisTypeSchema,
+      AnalysisTypeOptions options) {
+
+    List<String> fileTypes = new ArrayList<>();
+
+    if (options != null && CollectionUtils.isNotEmpty(options.getFileTypes())) {
+      fileTypes = options.getFileTypes();
+    }
     val analysisSchema =
-        AnalysisSchema.builder().name(analysisTypeName).schema(analysisTypeSchema).build();
+        AnalysisSchema.builder()
+            .name(analysisTypeName)
+            .schema(analysisTypeSchema)
+            .fileTypes(fileTypes)
+            .build();
+
+    log.info("file types " + fileTypes);
```

Comment:
This looks like developer test logs, I think we can remove this or change it to debug level and add more details from the analysis schema that is being created.